extensions/visualizations/oppia-visualization-sorted-tiles.directive.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the "sorted tiles" visualization.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+require('services/html-escaper.service.ts');
+
+angular.module('oppia').directive('oppiaVisualizationSortedTiles', [
+  'UrlInterpolationService',
+  function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {},
+      templateUrl: UrlInterpolationService.getExtensionResourceUrl(
+        '/visualizations/oppia-visualization-sorted-tiles.directive.html'),
```

Comment:
Can you try to use `template: require('visualizations/oppia-visualization-sorted-tiles.directive.html')` here?

Reply:
Sorry, can you try `require('./oppia-visualization-sorted-tiles.directive.html')`

Reply:
Cool, that worked! Thanks!